### ISSUE TITLE
Migration error right after installing Nova 4.13.0 "Call to a member function getKeyType() on null"

### ISSUE THREAD

**Issue by MartinVandersteen** [laravel/nova-issues#4764]
TLDR : 
**Nova 4.13.0**
fresh **Laravel 8.83.23** project
=> Throws "Call to a member function getKeyType() on null" when running migrations for the first time in file `2018_01_01_000000_create_action_events_table`

-----

Hello !

I just reinstalled Nova 4.13.0 in a fresh Laravel 8.83.23 project.
After running `php artisan nova:install` without issues, I get an error where running `php artisan migrate`.

The migration that triggers the error is `2018_01_01_000000_create_action_events_table`

Here is the error I'm getting :

```
Migrating: 2018_01_01_000000_create_action_events_table

   Error 

  Call to a member function getKeyType() on null

  at vendor/laravel/framework/src/Illuminate/Database/Schema/Blueprint.php:902
    898▕         if (is_string($model)) {
    899▕             $model = new $model;
    900▕         }
    901▕ 
  ➜ 902▕         return $model->getKeyType() === 'int' && $model->getIncrementing()
    903▕                     ? $this->foreignId($column ?: $model->getForeignKey())
    904▕                     : $this->foreignUuid($column ?: $model->getForeignKey());
    905▕     }
    906▕ 

      +28 vendor frames 
  29  artisan:37
      Illuminate\Foundation\Console\Kernel::handle()
```

In [this](https://www.rashadmirza.com/blog/11/the-best-laravel-admin-laravel-nova) blog post I found that you could just temporarily edit the code to add a `$model &&` before the problematic `->getKeyType()` call but I figured I would raise an issue since this should probably not happen and could confused less experienced developers.

Feel free to close for any reason as it is not blocking me anyways !

Thanks for the great tool and have a great day :)

**Comment by MartinVandersteen** [comment]
Ok, actually now that I dived a bit deeper into it I know I remember I had that problem once already.

For Statamic users : the problem is that in `config/auth.php`, in the `providers` array (line 68), Statamic sets a `users` provider without specifying a model. Nova expects a model to be defined in there.

Change this
```
'users' => [
      'driver' => 'statamic',
],
```

to this 
```
'users' => [
      'driver' => 'statamic',
      'model' => App\Models\User::class,
],
```

And you should be good !

**Comment by crynobone** [comment]
Yeah, it already been documented here: https://nova.laravel.com/docs/4.0/upgrade.html#eloquent-user-provider-required

**Comment by MartinVandersteen** [comment]
Oh I see, sorry, it doesn't come up when googling the issues but I guess this issue might help ! Sorry for the disturbance
